rasa/engine/graph.py

```diff
@@ -23,6 +23,8 @@
 class SchemaNode:
     """Represents one node in the schema."""
 
+    # TODO: JUZL: add name?
```

Comment:
@joejuzl You have this on your radar, right?

Reply:
I will remove this TODO as I don't think we need it.

Reply:
good to me